apps/glusterfs/cluster_entry.go

```diff
@@ -156,6 +156,11 @@ func (c *ClusterEntry) VolumeDelete(id string) {
 	c.Info.Volumes = sortedstrings.Delete(c.Info.Volumes, id)
 }
 
+// VolumeCount returns number of volumes in cluster *including* the pending ones
+func (c *ClusterEntry) VolumeCount() int {
```

Comment:
Can we start out by making this a private function? I don't think its needed outside of apps/glusterfs right?

Reply:
yes! Will do.